lib/model.ml

```diff
@@ -8,7 +8,18 @@ and entry_map = value_entry list KeyMap.t
 
 type t = Fix of t KeyMap.t
 
-let rec compare (Fix map1) (Fix map2) = KeyMap.compare compare map1 map2
+let rec is_empty (Fix t) =
+  KeyMap.is_empty t
+  || KeyMap.for_all (fun k v -> k = "" && is_empty v) t
```

Comment:
empty maps or maps with `""`→empty are all considered empty

it means that the following ccl document (e.g.,) is empty
```
=
  =
  =
=
=
```